package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "shortit-ts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon src/app.ts",
+    "tsc": "tsc"
```

You would need to install rimraf. Then ensure that when you run `yarn build`, it creates a `dist` folder. You'll need to update the tsconfig file to ensure the build output is called `dist`

```suggestion
    "build": "rimraf ./dist && tsc",
    "start": "yarn build && node dist/app.js",
    "dev": "nodemon src/app.ts",
    "lint": "eslint . --ext .ts,.js"
```